tests/api/test_datasets.py

```diff
@@ -3,24 +3,62 @@
 
 
 @pytest.mark.asyncio
-async def test_dataset_crud(client: httpx.AsyncClient) -> None:
-    response = await client.post("/datasets/", json={"name": "Example"})
+@pytest.mark.parametrize(
+    "payload, expected_errors_attrs",
+    [
+        pytest.param(
+            {},
+            [
+                {"loc": ["body", "title"], "type": "value_error.missing"},
+                {"loc": ["body", "description"], "type": "value_error.missing"},
+            ],
+            id="missing-fields",
+        ),
+    ],
+)
+async def test_create_dataset_invalid(
+    client: httpx.AsyncClient, payload: dict, expected_errors_attrs: list
+) -> None:
+    response = await client.post("/datasets/", json=payload)
+    assert response.status_code == 422
+
+    data = response.json()
+    assert len(data["detail"]) == len(expected_errors_attrs)
+
+    for error, expected_error_attrs in zip(data["detail"], expected_errors_attrs):
+        error_attrs = {key: error[key] for key in expected_error_attrs}
+        assert error_attrs == expected_error_attrs
+
+
+@pytest.mark.asyncio
+async def test_dataset_create(client: httpx.AsyncClient) -> None:
```

Comment:
very minor nit: here the test is named `test_dataset_create`, while the previous one is `test_create_dataset...`. Maybe use the same naming convention?

Reply:
Oui bien vu ! C'est une coquille

```suggestion
async def test_create_dataset(client: httpx.AsyncClient) -> None:
```